travharv/config_build.py

```diff
@@ -316,9 +317,9 @@ def _assert_valid_sparql_syntax(self, sparql_query):
 
     def _files_folder(self):
         return [
-            f
-            for f in os.listdir(self.config_files_folder)
-            if f.endswith((".yml", ".yaml"))
+            f.name
+            for f in Path(self.config_files_folder).iterdir()
```

Comment:
does iterdir recurse into subfolders like glob?
if not, would that not be nice to have?

Reply:
added rglob("*") that does do recursive calling